index.d.ts

```diff
@@ -39,6 +39,11 @@ export class Registry<
 	 */
 	resetMetrics(): void;
 
+	/**
+	 * Reset one naed metric
```

Comment:
`naed` seems like a typo? perhaps:
```suggestion
	 * Reset a named metric
	 * @param metric Metric to reset
```